lib/zomato.rb

```diff
@@ -0,0 +1,98 @@
+require 'net/http'
+
+$key = ENV['ZOMATO_API_KEY']
+$base_url = 'https://developers.zomato.com/api/v2.1'
+
+class Zomato
+
+  def initialize(city_id)
+    @city_id = city_id
+  end
+
+  def createCuisines
+    charset = Array(1..9) + Array('a'..'z')
+    uri = URI("#{$base_url}/cuisines?city_id=#{@city_id}")
+
+    Net::HTTP.start(uri.host, uri.port, :use_ssl => uri.scheme == 'https') do |http|
+      request = Net::HTTP::Get.new uri
+      request.add_field("user-key", $key)
+      request.add_field("accept", "application/json")
+      response = http.request request # Net::HTTPResponse object
+      if !response.is_a?(Net::HTTPSuccess)
+        puts "Failed status: #{response.code}"
+        return
+      end
+
+      JSON.parse(response.body)['cuisines'].each do |a|
+        Cuisine.create!({
+                            id: a['cuisine']['cuisine_id'],
+                            name: a['cuisine']['cuisine_name'],
+                            icon: charset.sample
```

Comment:
the whole `charset` here is a bit weird, but not really important

Reply:
It's for generating random icons for the cuisine.